great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -563,6 +574,48 @@ def _reconcile_rule_config(
 
         return effective_rule_config
 
+    @staticmethod
```

Comment:
@Shinnnyshinshin I am not seeing why this is needed.  Is there a problem with the existing `domain_builder_config: dict = rule_config.get("domain_builder", {})`?  Thanks.

Reply:
removed :)